src/fondant/cli.py

```diff
@@ -218,6 +202,9 @@ def start_explore(args):
         tag=args.tag,
         port=args.port,
         extra_volumes=extra_volumes,
+        auth_gcp=args.auth_gcp,
```

Comment:
What do you think about combining these arguments into a single one, for instance `auth_provider=AZURE,AWS,GCP`? Might be easier to update the different auth options without changing the interface.

Reply:
good suggestion! Implemented